byway/src/main/java/com/google/sps/data/Recommendation.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+import com.google.appengine.api.datastore.Entity;
+
+/** A sample class to mock data received from another datastore table*/
+// TODO: Make a parent class that Rec and Stop can extend 
+public final class Recommendation{
+  /** We may need to add more fields later, so making a class seemed reasonable. */
+  private final long id;
+  private final String placename; 
+  public static final String KIND = "Recommendation";
+  
+  public Recommendation(long id, String placename) {
+    this.id = id;
+    this.placename = placename;
+  }
+
+  public static Recommendation fromEntity(Entity entity){
+    long id = entity.getKey().getId();
```

Comment:
It would be good to add a `checkArgument(entity.getKind().equals(KIND))`, similar to what I did [here](https://github.com/googleinterns/step200-2020/blob/3ee1698046d63eca0ca7436e35aa88eed79d8d97/portfolio/src/main/java/com/google/sps/servlets/Comment.java#L76)

Reply:
Done.